parachain-template/runtime/src/lib.rs

```diff
@@ -516,7 +512,7 @@ impl pallet_xcm::Config for Runtime {
 	type XcmExecuteFilter = Everything;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 	type XcmTeleportFilter = Everything;
-	type XcmReserveTransferFilter = Everything;
+	type XcmReserveTransferFilter = ();
```

Comment:
```suggestion
	type XcmReserveTransferFilter = Nothing;
```

Wasn't there a `Nothing`?